Makefile

```diff
@@ -0,0 +1,542 @@
+# Copyright 2018  Google & GKE.
```

Comment:
AFAIK everything under github.com/kubernetes must be coyprighted to "The Kubernetes Authors"

Reply:
Good point.